teamfoundationclient.cpp

```diff
@@ -382,3 +405,26 @@ void TeamFoundationClient::addRecursive(QStringList &arguments, const QString &p
     if (fileInfo.isDir())
         arguments << QLatin1String("/recursive");
 }
+
+void TeamFoundationClient::configurationChanged()
+{
+    if (m_plugin->settings().binaryPath().isEmpty()) {
+        m_tfVersion = TfVersion_None;
+    } else {
+        TeamFoundationResponse response = runTf(QStringLiteral("C:/"), QStringList(), SuppressCompletely);
+        QRegularExpression re(QStringLiteral("Version (\\d+)"));
+        QRegularExpressionMatch match = re.match(response.standardOut);
+        if (match.hasMatch()) {

```

Comment:
That is a good point. I suppose one could argue that it is possible to place a break point on the line in case there is no TF binary.